### ISSUE TITLE
s

### ISSUE THREAD

**Issue by memeplex** [orgzly/orgzly-android#442]
I have searched for existing issues that may be the same as or related to mine. 👼

I'm submitting a **bug report** for version REPLACE-WITH-APP-VERSION.
I'm submitting a **feature request**.

**Comment by nevenz** [comment]
You can currently share to Orgzly which creates a new note with the link.

Are you suggesting sharing to *existing* note?

**Comment by memeplex** [comment]
No, sharing to the clipboard formatted as an org link so that it's easily pasteable into an existing note.

**Comment by nevenz** [comment]
Perhaps a (new) button for inserting link could check the clipboard data and use it?

Toolbar with various buttons (bold, italic, insert link) is planned for when note's title or content are being edited.

**Comment by memeplex** [comment]
That would be great, but even standard paste would be far better than
having to type `[[][]]` move the pointer back, paste, move the pointer
back, copy&paste.

**Comment by memeplex** [comment]
Btw @nevenz what are the plans for the toolbar? Are you working on that?

**Comment by nevenz** [comment]
> Btw @nevenz what are the plans for the toolbar? Are you working on that?

I was working on it but moved to other things. I have some code around.

**Comment by memeplex** [comment]
I notice I forgot to explain the use case for this. Maybe it's not as evident as I believe.

The scenario is that you have already written a note or are writing a somewhat large one, with two or more links. Then you need to inline links instead of making them subitems of the current node. Insead of pasting an ugly bare url you want to paste a nicely formatted org link. I've used a couple of android markdown note taking apps that let you capture a mardown formatted url in the clipboard so you're then able to paste it at your whim.
